code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1867,6 +1872,10 @@
 	return 0
 
 /mob/living/carbon/human/proc/can_drink(var/obj/item/I)
```

Comment:
Adding the check for the reactor right at the start and returning TRUE, prevents the other checks from running (i.e. if the IPC has a mouth and if the mouth is covered)

Reply:
intended as baselines and etc don't have a mouth and thus cannot eat